app/models/adjusted_fee.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AdjustedFee < ApplicationRecord
+  belongs_to :invoice
+  belongs_to :fee, optional: true
```

Comment:
Why is the fee optional?

Reply:
The reason behind was that I wanted to ensure that this object can exist without fee (in scenario when refresh job is called and fee got destroyed)